### DISCUSSION TITLE
Adding DSCP and MPLS label(s) value in dimensions

### DISCUSSION THREAD

**Discussion by edonie** [akvorado/akvorado#558] [Q&A]
Hi everyone,

Is it possible to add DSCP (IP TOS in netflow template) value in dimensions? How about MPLS labels also? If this is indeed possible can someone please provide me with some instructions it would be greatly appreciated.

**Comment by vincentbernat** [comment]
Both of them is currently unimplemented. See https://github.com/akvorado/akvorado/commit/9eee46caded6fa6cc2dfb90d1941b2ef05d15e9f for an example on how to do that.

**Reply by edonie** [reply]
Thank you Vincent!

**Reply by azamatmunalbay** [reply]
@vincentbernat Could you please share whether adding DSCP (IP TOS) and MPLS label fields to dimensions is planned for future Akvorado releases or roadmap?
We’re currently evaluating flow telemetry visibility and this feature would be very useful for QoS analysis.

Thanks again for your work on Akvorado!
